### PR TITLE
MULTIARCH-3741 - Updating the 4.10 through 4.14 periodic cron configs

### PR DESCRIPTION
**Pull request by tvardema**
Updating the cron definitions for each of the ppc64le and s390x jobs to alleviate potential resource issues, and have a more predictable periodic "calendar."

### CODE REVIEW

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml [hunk 1]
```diff
@@ -233,7 +233,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-image-ecosystem-remote-libvirt-s390x
   cluster: build01
-  cron: 5 1 7 */12 *
+  cron: 0 0 * * 0
```
Comment (Prashanth684) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml:236]: for future reference - do you want to add a comment on what this translates to ?

Comment (tvardema) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml:236]: Yes, I do.  I'll update with human-readable descriptions in comments.  Good call.

Comment (tvardema) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml]: Unfortunately the comments are removed when running `make update`.  I think we'll have to go without :(

**Pull request merged**